packages/bitcore-node/src/providers/chain-state/evm/api/csp.ts

```diff
@@ -106,16 +106,31 @@ export class BaseEVMStateProvider extends InternalStateProvider implements IChai
   }
 
   async getFee(params) {
-    let { network, target = 4 } = params;
+    let { network, target = 4, txType, priorityFeePercentile } = params;
```

Comment:
We should decouple the priority fee from this method altogether and make another method called like `getPriorityFee`. I don't think we need one mega function that handles all the things...having separate functions that are called in their own contexts would be ideal, I think